tests/integration/ssr/fixtures/partial/src/one/routes/b/d/page.data.ts

```diff
@@ -0,0 +1,4 @@
+export const loader = () => {
+  console.log(document.querySelector);
```

Comment:
Does we must need this console?

Reply:
Yeah, I want to use it ensure `page.data.ts` always run on the browser

Reply:
Got it